javascript/safari-driver/inject/build.desc

```diff
@@ -38,6 +38,7 @@ js_module(
     srcs = [
         "inject.js",
         "tab.js",
+        "html2canvas.js",
```

Comment:
Create a new module for this (and the associated command). Maybe `js_module(name = "canvas")`